index.html

```diff
@@ -0,0 +1,611 @@
+              <figure class="app-review-media">
+                <img src="/img/review-image-placeholder.png" alt="Image of PLACEHOLDER" title="PLACEHOLDER">
+                <figcaption>
+                  <h2>Nick "Goose" Bradshaw</h2>
+                  <p>LTJG, Radar Intercept Officer</p>
+                </figcaption>
+              </figure>
+            </li>
+          </ul>
+          <div class="swiper-pagination swiper-pagination-1"></div>
+        </div>
+      </section>
+      <section data-section="appbox-screenshots" id="screens">
+        <h1 class="header-underline">AppBox Screenshots</h1>
+        <ul class="l-flex-container">
+          <li>
+            <figure>
+              <img src="/img/iphone-screenshot.svg" alt="Screenshot 1" title="Screenshot 1">
+            </figure>
+          </li>
+          <li class="screenshot">
+            <figure>
+              <img src="/img/iphone-screenshot.svg" alt="Screenshot 2" title="Screenshot 2">
+            </figure>
+          </li>
+          <li class="screenshot">
+            <figure>
+              <img src="/img/iphone-screenshot.svg" alt="Screenshot 3" title="Screenshot 3">
+            </figure>
+          </li>
+        </ul>
+        <a href="#"><img src="/img/left-arrow-screenshot.svg"></a>
+        <a href="#"><img src="/img/right-arrow-screenshot.svg"></a>
+      </section>
+      <section data-section="more-apps" id="more">
+        <h1 class="header-underline">More Apps By AppBox</h1>
+        <div class="swiper-container swiper-2">
+          <ul class="swiper-wrapper swiper-wrapper-2">
+            <li class="app-store-item-preview swiper-slide">
+              <figure>
+                <img src="/img/app-store-placeholder.png" alt="App Store Placeholder Image" title="App Store Placeholder Image">
+                <p class="review-text-overlay">Lorem ipsum dolor sit amet consectetur adipisicing elit sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam quis etu nostrud exercitation ullamco laboris</p>
+                <figcaption>
+                  <h2>Sports iPhone App</h2>
+                  <img src="/img/orange-star.svg" alt="positive star rating" title="Positive Star Rating">
+                  <img src="/img/orange-star.svg" alt="positive star rating" title="Positive Star Rating">
+                  <img src="/img/orange-star.svg" alt="positive star rating" title="Positive Star Rating">
+                  <img src="/img/orange-star.svg" alt="positive star rating" title="Positive Star Rating">
+                  <img src="/img/grey-star.svg" alt="negative star rating" title="Negative Star Rating">
+                  <p>$5.00</p>
+                </figcaption>
```

Comment:
There are a number of ways this cat could have been skinned; unfortunately you chose the wrong one. This is a text book example of the [`<meter>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meter).

A component like this could be as simple as:

```html
<meter max="5"  min="0" value="4"></meter>
```

```css
/* ... other meter ratings ... */
meter[value="4"] {
  background: url('img/meter-4-star.svg');
}
/* ... other meter ratings ... */
```

All the way up to something like this: http://codepen.io/DonSinDRom/pen/YPKdwB